src/main/resources/assets/chiseled_me/lang/fr_fr.lang

```diff
@@ -0,0 +1,65 @@
+
+itemGroup.chiseled_me=Chiseled Me
+
+chiseled_me.bed.too_small=Vous êtes trop petit pour dormir dans un lit !
+chiseled_me.bed.too_big=Vous êtes trop petit pour dormir dans un lit !
+
+commands.chiseled_me:sizeof.usage=/sizeof <entity> [(set|add|subtract|multiply|divide) <size> [animate [<ticks>]]]
+commands.chiseled_me:sizeof.get=La taille relative de %s est %s!
+commands.chiseled_me:sizeof.set=Réglage de la taille de %s à %s!
+
+item.chiseled_me:recalibrator.charges=  §8Charges restantes: §7%s
+item.chiseled_me:recalibrator.reset.name=Recalibrateur Volumétrique de Masse [ 1.0 ]
+item.chiseled_me:recalibrator.reset.tooltip=§8§o-Clic droit pour rétablir votre taille normale.
+item.chiseled_me:recalibrator.reduction.name=Recalibrateur Volumétrique de Masse [ 1/%s ]
+item.chiseled_me:recalibrator.reduction.tooltip=§8§o-Clic droit pour être  1/%s ème de votre taille  normale.
+item.chiseled_me:recalibrator.amplification.name=Recalibrateur Volumétrique de Masse [ %s ]
+item.chiseled_me:recalibrator.amplification.tooltip=§8§o-Clic droit pour être %s fois plus grand que votre taille normale.
+
+item.chiseled_me:blue_star.name=Étoile du Nether Bleue
+item.chiseled_me:pym_container.name=Conteneur de Pym
+item.chiseled_me:pym_container_x.name=Conteneur de Pym Renforcé
+item.chiseled_me:pym_essence.name=Essence de Pym
+item.chiseled_me:pym_essence_x.name=Essence de Pym Concentrée
+item.chiseled_me:pym_essence_b.name=Essence de Pym Concentrée Antipolarisée
+
+advancement.chiseled_me:beginning=Le début
+advancement.chiseled_me:beginning.desc=Fabriquer un conteneur de pym
```

Comment:
shouldn't it be Pym here?

Reply:
It wasn't capitalized on the english source, if that's what you mean, so if it's supposed to be P instead of p, need to fix that on the english file too

Reply:
yeah ok I'll just fix it everywhere myself then